docs/dialect.md

```diff
@@ -173,3 +173,99 @@ Allowable date parts: year, quarter, month, dayofyear, day, weekday, week, hou
 
   **Join Capabilities**  <br/>
   Join usage is also defined by capabilities in the manifest file. See the Query section for join relation capabilities [here]({{ site.baseurl }}/docs/capabilities).
+
+### Boolean Support:
+  Some databases need to customize boolean support functions.  A common case is when a database lacks native boolean support. <br/>
+   **format-true** <br/> 
+  String literal or expression for boolean false value.  
+  ```xml 
+   <format-true value='(1=1)' />
+  ```
+'format-true' and 'format-false' are used in two use cases: literals and predicates
```

Comment:
Move this above format-true and update it to use 
```suggestion
'format-true' and 'format-false' are used in predicate statements
```

Reply:
Removed the mention of literal support and added `'format-true' and 'format-false' are used in predicate statements` in the documentation